concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-42	42	Morgen	Morgen	morning	1339	MORNING
+Steinthal-1875-1565-43	43	Mittag	Mittag	noon	12	MIDDAY
+Steinthal-1875-1565-44	44	Abend	Abend	evening	1629	EVENING
+Steinthal-1875-1565-45	45	Morgenroth	Morgenroth	dawn	729	DAWN
+Steinthal-1875-1565-46	46	Abendröthe	Abendröthe	dusk	2297	DUSK
+Steinthal-1875-1565-47	47	Es ist finster.	Es ist finster.	It is gloomy.	1552	GLOOMY
+Steinthal-1875-1565-48	48	Finsterniss	Finsternis	gloom		
+Steinthal-1875-1565-49	49	dunkel, -heit	dunkel	dark	706	DARK
+Steinthal-1875-1565-50	50	dunkel, -heit	Dunkelheit	darkness	11	DARKNESS
+Steinthal-1875-1565-51	51	hell	hell	bright	679	BRIGHT
+Steinthal-1875-1565-52	52	Licht	Licht	light	630	LIGHT (RADIATION)
+Steinthal-1875-1565-53	53	Schatten	Schatten	shadow	1388	SHADE
+Steinthal-1875-1565-54	54	Jahr	Jahr	year	1226	YEAR
+Steinthal-1875-1565-55	55	Jahreszeit	Jahreszeit	season	691	SEASON
+Steinthal-1875-1565-56	56	Sommer (trockne Zeit)	Sommer	summer	1342	SUMMER
+Steinthal-1875-1565-57	57	Sommer (trockne Zeit)	trockne Zeit	dry season	452	DRY SEASON
+Steinthal-1875-1565-58	58	Winter (Regenzeit)	Winter	winter	1471	WINTER
+Steinthal-1875-1565-59	59	Winter (Regenzeit)	Regenzeit	wet season	453	RAINY SEASON
+Steinthal-1875-1565-60	60	- ist nun vorüber.	Sommer, Winter ist nun vorüber.	Summer, winter is over now.		
+Steinthal-1875-1565-61	61	Herbst (Erntezeit)	Herbst	autumn	1276	AUTUMN
+Steinthal-1875-1565-62	62	Herbst (Erntezeit)	Erntezeit	harvest season	3327	AUTUMN OR HARVEST SEASON
+Steinthal-1875-1565-63	63	Frühling (Saatzeit)	Frühling	spring	1537	SPRINGTIME
+Steinthal-1875-1565-64	64	Frühling (Saatzeit)	Saatzeit	sowing time		
+Steinthal-1875-1565-65	65	heute	heute	today	1283	TODAY
+Steinthal-1875-1565-66	66	gestern	gestern	yesterday	1174	YESTERDAY
+Steinthal-1875-1565-67	67	vorgestern	vorgestern	day before yesterday	1180	DAY BEFORE YESTERDAY
+Steinthal-1875-1565-68	68	morgen	morgen	tomorrow	1329	TOMORROW
+Steinthal-1875-1565-69	69	übermorgen	übermorgen	day after tomorrow	1179	DAY AFTER TOMORROW
+Steinthal-1875-1565-70	70	Woche	Woche	week	1469	WEEK
+Steinthal-1875-1565-71	71	Monat	Monat	month	1370	MONTH
+Steinthal-1875-1565-72	72	Namen der Wochentage und der Monate	Namen der Wochentage und der Monate	names of weekdays and months		
+Steinthal-1875-1565-73	73	die Weltgegenden	Norden	north	1598	NORTH
+Steinthal-1875-1565-74	74	die Weltgegenden	Osten	east	1877	EAST
+Steinthal-1875-1565-75	75	die Weltgegenden	Süden	south	1595	SOUTH
+Steinthal-1875-1565-76	76	die Weltgegenden	Westen	west	1817	WEST
+Steinthal-1875-1565-77	77	die Weltgegenden	Nordosten	northeast		
+Steinthal-1875-1565-78	78	die Weltgegenden	Südosten	southeast		
+Steinthal-1875-1565-79	79	die Weltgegenden	Nordwesten	northwest		
+Steinthal-1875-1565-80	80	die Weltgegenden	Südwesten	southwest		
+Steinthal-1875-1565-81	81	Es ist kalt.	Es ist kalt.	It is cold.	2483	COLD (OF WEATHER)
+Steinthal-1875-1565-82	82	Kälte	Kälte	cold	2932	COLD (CHILL)
```

Comment:
Why this particularly mapping for this and the cold before that?

Reply:
81: You are right, I interpreted it as talking about the weather, but could really be about anything. I will map it to COLD instead.
82: Here, I went for COLD (CHILL) because this is the concept that refers to Person/Thing instead of Property. Dellert-2017-1016 also has a German Kälte mapped to this concept.